server/conf/env-var-docs.json

```diff
@@ -455,7 +450,7 @@
     ]
   },
   "CIVIFORM_SUPPORTED_LANGUAGES": {
-    "mode": "ADMIN_READABLE",
+    "mode": "HIDDEN",
```

Comment:
Since it's a build-in I can't get rid of the nested namespace for it in HOCON (`play.i18n.`) which makes it hard to correlate it with an env var. I'm still thinking about how to address this holistically, for now just leaving out this config var.

Reply:
FWIW the list of supported languages for a given deployment is visible to the admin whenever they edit the translations for a question or program.